src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -238,5 +176,18 @@ public static Metric HandledException(string exceptionName, string connectorStag
                     { _operationDimension, connectorStage },
                 });
         }
+
+        private static Metric CreateBaseIoMTMetric(IomtMetricNames metricName, string category, string operation)
```

Comment:
>CreateBaseIoMTMetric [](http://example.com/codeflow?start=30&length=20)

Suggestion:  You move this as an extension in the MetricExtensions and get a more fluent style syntax.